src/server/scripts/Commands/cs_debug.cpp

```diff
@@ -481,39 +452,31 @@ class debug_commandscript : public CommandScript
         return true;
     }
 
-    static bool HandleDebugSendChannelNotifyCommand(ChatHandler* handler, char const* args)
+    static bool HandleDebugSendChannelNotifyCommand(ChatHandler* handler, ChatNotify type)
     {
-        if (!*args)
-            return false;
-
         char const* name = "test";
-        uint8 code = atoi(args);
 
         WorldPacket data(SMSG_CHANNEL_NOTIFY, (1+10));
-        data << code;                                           // notify type
-        data << name;                                           // channel name
+        data << uint8(type);
```

Comment:
`static_cast` (here and everywhere)

Reply:
Hmm, that's the style we use in the packet classes, so I would leave this as is.